Source/Zencoder/HttpPostNotificationOutput.cs

```diff
@@ -134,5 +134,23 @@ public class HttpPostNotificationOutput
         /// </summary>
         [JsonProperty("type")]
         public OutputType OutputType { get; set; }
+
+        /// <summary>
+        /// error type, eg "DownloadAccessDeniedError"
+        /// </summary>
+        [JsonProperty("error_class")]
+        public string ErrorClass { get; set; }
+
+        /// <summary>
+        /// Long description of error for human consumption
+        /// </summary>
+        [JsonProperty("error_message")]
+        public string ErrorMessage { get; set; }
+
+        /// <summary>
+        /// url for help page with more information about the error
+        /// </summary>
+        [JsonProperty("error_link")]
+        public string ErrorLink { get; set; }
```

Comment:
I think it would be a better design to have a separate error notification. What do you think?

Reply:
Pretext: looks like the error handling is already borked. If we separate the error notification we wont alter the definition of the current output. Any other consumer won't be affected as they will have to intentionally subscribe to this new type.

Reply:
Unfortunately we can't control the structure of this object, Zencoder has defined it and we have to handle what they send us